pages/docs/doc/fipe.json

```
@@ -157,6 +157,65 @@
                     }
                 }
             }
+        },
+        "/fipe/veiculos/v1/{tipoVeiculo}": {
+            "get": {
+                "tags": ["FIPE"],
+                "summary": "Lista os veículos de acordo com a marca e o tipo de veículo",
+                "description": "",
+                "parameters": [
+                    {
+                        "name": "tipoVeiculo",
+                        "description": "Os tipos suportados são `caminhoes`, `carros` e `motos`.\n",
+                        "in": "path",
+                        "required": false,
+                        "schema": {
+                            "type": "string",
+                            "format": "string"
+                        }
+                    },
+                    {
+                        "name": "codigoMarca",
+                        "description": "Código da marca do veiculo.\n",
```

Talvez seja interessante dar uma dica de onde encontrar as marcas
```suggestion
                        "description": "Código da marca do veiculo. Para consultar as marcas acesse a rota `/fipe/marcas/v1/`",
```

---

documentação atualizada.